### ISSUE TITLE
Migrate to Postgresql

### ISSUE THREAD

**Issue by ndelvalle** [ndelvalle/rustapi#408]

**Comment by opeolluwa** [comment]
Do you want help with this?

**Comment by ndelvalle** [comment]
@opeolluwa Yes, but I still haven’t decided what to use for PostgreSQL. I’m leaning more towards Diesel over SeaORM—it seems simpler and doesn’t clutter the code with ORM-related elements everywhere. I also need to put more effort into my #458 PR to make it more stable so that others can contribute, as it’s currently a bit messy.
Do you have strong opinions about these ORMs?

**Comment by opeolluwa** [comment]
yes, I've used both in production environment. 
My opinion is that, diesel is more battle tested than SeaORM, especially looking at the stats, on the other end, SeaORM seems more promising.

Also on the other end, Diesel isn't async-first unlike, SeaORM.

**Comment by opeolluwa** [comment]
Do you want me to look at this

**Comment by ndelvalle** [comment]
@opeolluwa I’m currently using diesel-async for async support, and it seems to be working correctly. If you’re interested in continuing the work, that would be fantastic. I haven’t had much time lately to make progress, so any help would be greatly appreciated!

**Comment by opeolluwa** [comment]
I'll see what can be done

**Comment by opeolluwa** [comment]
@ndelvalle do you want to use diese-async or I should revamp with sea-orm

**Comment by ndelvalle** [comment]
@opeolluwa i think using async diesel is the best idea.

**Comment by opeolluwa** [comment]
alright

**Comment by opeolluwa** [comment]
@ndelvalle  would you be kind to tag the branch you're working on

**Comment by ndelvalle** [comment]
> @ndelvalle would you be kind to tag the branch you're working on

I just added you as a collaborator, feel free to keep working on the same branch or tag it.

**Comment by opeolluwa** [comment]
awesome thanks

**Comment by opeolluwa** [comment]
@ndelvalle  the use-diesel branch has a handful of issues, I'm going to squash it and incrementally make PR to the main branch till all it is all setup

**Comment by ndelvalle** [comment]
> @ndelvalle the use-diesel branch has a handful of issues, I'm going to squash it and incrementally make PR to the main branch till all it is all setup

Yeah, we don't need to use the use-diesel branch, you can create another one and we can work on that one.

**Comment by opeolluwa** [comment]
Awesome! 

I've done that
